company_multi_logo/README.rst

```diff
@@ -0,0 +1,56 @@
+Company Multiple Logo
```

Comment:
```
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License
```
according to OCA's template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst